Scripts/Data/Type/Transformation/FloatAdder.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace VRTK.Core.Data.Type.Transformation
+{
+    using UnityEngine.Events;
+    using System;
+    using System.Linq;
+
+    /// <summary>
+    /// Processes a given <see cref="float"/> and adds it to all other values in the collection and outputs the sum of all values.
```

Comment:
This seems to be no longer true. What about this instead:
```suggestion
    /// Sums up a collection of <see cref="float"/>s.
```